integration/container/create_test.go

```diff
@@ -70,6 +71,74 @@ func TestCreateFailsWhenIdentifierDoesNotExist(t *testing.T) {
 	}
 }
 
+func TestCreateByImageID(t *testing.T) {
+	ctx := setupTest(t)
+	apiClient := testEnv.APIClient()
+
+	img, _, err := apiClient.ImageInspectWithRaw(ctx, "busybox")
+	assert.NilError(t, err)
+
+	imgIDWithAlgorithm := img.ID
+	imgID, _ := strings.CutPrefix(imgIDWithAlgorithm, "sha256:")
+	imgShortID := stringid.TruncateID(imgID)
+
+	testCases := []struct {
+		doc             string
+		image           string
+		expectedErrType func(error) bool
+		expectedErr     string
+	}{
+		{
+			doc:   "image ID with algorithm",
+			image: imgIDWithAlgorithm,
+		},
+		{
+			doc:   "image ID without algorithm",
+			image: imgID,
```

Comment:
Looks like this  is  testing the same  as https://github.com/moby/moby/blob/58125b6df64fb34bdee5f073febcbda9f3b02fbf/integration-cli/docker_cli_create_test.go#L306-L312


Let me remove that  test, and add a comment here that  it's related  to https://github.com/moby/moby/issues/20972

Reply:
☝️ done 👍